robot_skills/src/robot_skills/sergio.py

```diff
@@ -10,11 +26,31 @@ def __init__(self, wait_services=False):
 
         self._ignored_parts = ["leftArm", "rightArm", "torso", "spindle", "head"]
 
-        # This is still very ugly, because there is a lof of double code, but atleast it is only in sergio.
-        self.parts['leftArm'] = robot.arms.FakeArm(self.robot_name, self.tf_listener, side="left")
-        self.parts['rightArm'] = robot.arms.FakeArm(self.robot_name, self.tf_listener, side="right")
+        self.add_body_part('base', base.Base(self.robot_name, self.tf_listener))
+        self.add_body_part('torso', torso.Torso(self.robot_name, self.tf_listener))
+
+        self.add_body_part('leftArm', arms.FakeArm(self.robot_name, self.tf_listener, side="left"))
```

Comment:
Can you make an issue to remove this eventually?

Reply:
Done #706